src/components/TabLinks.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from "react";
+import { Tabs } from "@canonical/react-components";
+import { useNotify } from "@canonical/react-components";
+import { useNavigate } from "react-router-dom";
+import { slugify } from "util/slugify";
+
+interface Props {
+  TABS: string[];
+  activeTab?: string;
+  tabUrl: string;
+}
+
+const TabLinks: FC<Props> = ({ TABS, activeTab, tabUrl }) => {
+  const notify = useNotify();
+  const navigate = useNavigate();
+
+  const TABS_PATH = TABS.map((tab) => slugify(tab));
+
+  const handleTabChange = (tabUrl: string, newTab: string) => {
+    notify.clear();
+    if (newTab === TABS_PATH[0]) {
+      navigate(tabUrl);
+    } else {
+      navigate(`${tabUrl}/${newTab}`);
+    }
+  };
+
+  return (
+    <Tabs
+      links={TABS.map((tab, index) => {
+        const tabPath = TABS_PATH[index];
+
+        return {
+          label: tab,
+          id: tabPath,
+          active: tabPath === activeTab || (tab === TABS[0] && !activeTab),
+          onClick: (e) => {
+            e.preventDefault();
+            handleTabChange(tabUrl, tabPath);
+          },
+          href: tabPath === TABS[0] ? tabUrl : `${tabUrl}/${tabPath}`,
```

Comment:
Can we put this calculation into line 32, something like:
```
const href=tabPath === TABS[0] ? tabUrl : `${tabUrl}/${tabPath}`
```

Then we can use `href` directly.
And we can remove the `handleTabChange` function. Move the two lines `navigate(href)` and `notify.clear()` into the `onClick` handler.
This avoids the duplicate calculation of the href and should simplify the component a little bit.

Reply:
Good idea 👍